yellow-paper/docs/cryptography/protocol-overview.md

```diff
@@ -0,0 +1,113 @@
+# Proving System Components
+
+# Interactive Proving Systems
+
+## Ultra Plonk
+
+UltraPlonk is a variant of the [PLONK](https://eprint.iacr.org/2019/953) protocol - a zkSNARK with a universal trusted setup.
+
+UltraPlonk utilizes the "Ultra" circuit arithmetisation. This is a configuration with four wires per-gate, and the following set of gate types:
+
+- arithmetic gate
+- elliptic curve point addition/doubling gate
+- range-check gate
+- plookup table gate
+- memory-checking gates
+- non-native field arithmetic gates
+
+## Honk
+
+Honk is a variant of the PLONK protocol. Plonk performs polynomial testing via evaluating a polynomial relation is zero modulo the vanishing polynomial of a multiplicative subgroup. Honk performs the polynomial testing via evaluating, using a sumcheck protocol, that a relation over multilinear polynomials vanishes when summed over a boolean hypercube.
```

Comment:
by checking instead of via evaluating seems more readable to me

Reply:
updated

Reply:
![honk](https://github.com/AztecProtocol/aztec-packages/assets/304108/c586cfa1-287c-42a8-8feb-8078d19c63bd)